### DISCUSSION TITLE
請問老師或各位大大要怎麼刪除命令提示字元裡的這段文字呢?

### DISCUSSION THREAD

**Discussion by luo910227** [1101-GS4719-C/Python-Community-GS4719-C#3] [Q&A] [answered]
![1631775977542](https://user-images.githubusercontent.com/90717869/133566089-a5c42197-d019-43fa-a5bb-e1874d33337b.jpg)
如圖，上課的時候太急就少打了一個r，然後打成了老師的名字，後來就多了這一行，不知道要如何刪除了~麻煩會的人留言一下，萬分感謝

**Comment by chia-kai-chang** [comment, marked as answer]
再打一次：
`git config --global user.name 輸入自己的名字`
email也是可以再打一次：
`git config --global user.email 輸入自己的email`

**Reply by luo910227** [reply]
謝謝老師！那請問這一行有刪除的必要嗎？

**Reply by chia-kai-chang** [reply]
哈哈，終於懂你發問的問題了，使用這一行指令試試看：
`git config --global --unset use.name`

參考來源：https://stackoverflow.com/questions/11868447/how-can-i-remove-an-entry-in-global-configuration-with-git-config

**Reply by luo910227** [reply]
學到了！謝謝老師的解答，原來可這樣打🙏👍

**Comment by enoch-lightning** [comment]
是個好問題，我自己沒有遇到。雖然現在我連這個問題是什麼都不清楚，但也可以做為日後參考的資料。感謝同學的提問與老師的回答。

**Comment by chupochun** [comment]
謝謝同學的提問 讓我知道以後遇到這個問題的解決方法

**Comment by pekoranoinu** [comment]
這個部份我現在還沒用到，不過當我遇到的時候就有辦解決了。
